src/java.base/share/classes/java/text/MessageFormat.java

```diff
@@ -82,8 +82,16 @@
  *         { <i>ArgumentIndex</i> , <i>FormatType</i> }
  *         { <i>ArgumentIndex</i> , <i>FormatType</i> , <i>FormatStyle</i> }
  *
- * <i>FormatType: one of </i>
- *         number date time choice
+ * <i>FormatType:</i>
+ *         number
+ *         dtf_date
+ *         dtf_time
+ *         dtf_datetime
+ *         <i>pre-defined DateTimeFormatter(s)</i>
```

Comment:
I moved these new types before the existing formats because I wanted them

1. To be grouped with the other date/time related formats
2. To appear before the existing the other date/time related formats, as java.time takes precedence over util.Date

However, I can update it so that the new types/table are chronologically ordered; please let me know if you would prefer that.